include/aspect/parameters.h

```diff
@@ -323,7 +324,14 @@ namespace aspect
     unsigned int                   n_expensive_stokes_solver_steps;
     double                         temperature_solver_tolerance;
     double                         composition_solver_tolerance;
-
+    unsigned int                   max_pre_newton_nonlinear_iterations;
+    unsigned int                   max_newton_line_search_iterations;
+
+    /// temporary place it in parameters.h till a better place is found
```

Comment:
I see. Do you foresee them being used in more than one place? If not, then just define them in the code where they are used.